cpu/esp32/periph/adc.c

```diff
@@ -167,6 +159,8 @@ static bool _adc2_ctrl_initialized = false;
 #endif /* defined(ADC_GPIOS) || defined(DAC_GPIOS) */
 
 #if defined(ADC_GPIOS)
+static const unsigned adc_chn_num = ADC_NUMOF;
```

Comment:
If we now have the macro, why define `adc_chn_num`?

Reply:
Good question. It was just because I got compile errors when ADC_NUMOF became 0. I fixed it.